packages/main/src/themes/StepInput.css

```diff
@@ -150,15 +150,16 @@
 	vertical-align: top;
 	margin-top: var(--_ui5_step_input_input_margin_top);
 	min-width: var(--_ui5_step_input_min_width);
+	padding-left: var(--_ui5_step_input_padding_left);
```

Comment:
Here because the paddings are from both the side and If they are symmetric, it won't be a problem, because even if not switched, it's fine, but still we prefer the usage of css logical props